pkg/actor/deploy.go

```diff
@@ -50,6 +53,23 @@ func (d deploy) Act(ctx context.Context, cluster *resource.Cluster) error {
 
 	r := resource.NewManagedKubeResource(ctx, d.client, cluster, kube.AnnotatingPersister)
 
+	if cluster.Status() == nil || cluster.Status().Conditions.GetCondition(api.ConditionInstalling) == nil {
```

Comment:
Status gets set to installing if it's already installing?

Will this happen in the case of an error when installing?

Reply:
We probably should do an error state.  Will file an issue to get an error state created.